admin_manual/configuration_user/instruction_set_for_users.rst

```diff
@@ -155,9 +155,10 @@ XML output
 Edit data of a single user
 --------------------------
 
-Edits attributes related to a user. Users are able to edit email, displayname 
-and password; admins can also edit the quota value. Authentication is done by 
-sending a Basic HTTP Authorization header.
+Edits attributes related to a user. Users are able to edit email, displayname
+and password; admins can also edit the quota value. Further restrictions may apply,
+check the :ref:`Editable data fields list <_editable_field_list>`. Authentication
```

Comment:
does the link work? the github action annotation says no

Reply:
Right, whatever went wrong there... fixed now. Also built locally to confirm